Theory_assignment_two/assignment_2.md

```diff
@@ -0,0 +1,453 @@
+﻿
+## 1. How does Object Oriented Programming differ from Process Oriented Programming? 
+
+Procedural oriented programming is based on subroutines or procedure calls. Consist of sequential steps to perform specified tasks.
```

Comment:
9/10 great comparison and difference between POP and OOP.
Missing point: POP not divided into objects but functions